modules/imaging_browser/README.md

```diff
@@ -0,0 +1,81 @@
+# Imaging Browser
+
+## Purpose
+
+The imaging browser is intended to allow users to view candidate
+scan sessions collected for a study.
+
+## Intended Users
+
+The three primary types of users are:
+1. Imaging specialists using the modules to do online QC
+2. Project radiologists viewing images to report incidental findings
+3. Site coordinators or researchers ensuring their uploaded scans have
+   been processed and inserted into LORIS.
+
+## Scope
+
+The imaging browser displays images processed images which meet the
+study's inclusion criteria. The inclusion criteria (for most images)
+is defined and enforced in the LORIS imaging pipeline scripts.  Derived
+or processed scan types are also included and have their own insertion
+criteria.
+
+NOT in scope:
+
+The imaging browser module does not display raw DICOMs or perform automated
+quality control on images. It only displays images that have already been
+inserted into LORIS.
+
+## Permissions
+
+The imaging browser module uses the following permissions. Any one of them
+(except imaging_browser_qc) is sufficient to have access to the module.
+
+imaging_browser_view_allsites
+    - This permission gives the user access to all scans in the database
+
+imaging_browser_view_site
+    - This permission gives the user access to scans from their own site only
+
+imaging_browser_phantom_allsites
+    - This permission gives the user access to phantom, but not candidate, scans
+      across the database
+
+imaging_browser_phantom_ownsite
+    - This permission gives the user access to phantom, but not candidate, data
+      only at the user's site.
+
+imaging_browser_qc
+    - This permission gives the user access to modify the quality control data
+      for the associated scans and timepoints.
+
+## Configurations
+
+The imaging browser has the following configurations that affect its usage
+
+tblScanTypes - This setting determines which scan types are considered "NEW" for
+        QC purposes.
```

Comment:
and it also dictates the columns displayed in the Imaging Browser table page as well (as these columns are now dynamically generated based on these tblScanTypes).

Reply:
I added a sentence, how does it look now?